build.sh

```diff
@@ -497,14 +497,14 @@ class Maker:
                 query = "SELECT name from users WHERE uid=1"
                 uid1_name = self._drush(
```

Comment:
Isn't it actually the opening bracket here that is missing? The first parameter to _drush should be an array of drush commandline parameters.